mod.test.ts

```diff
@@ -248,15 +248,16 @@ Deno.test("should handle boolean list 'and'", async () => {
 
 Deno.test("should support custom command handlers", async () => {
   const builder = new CommandBuilder()
+    // deno-lint-ignore require-await
     .registerCommand("zardoz-speaks", async (context) => {
```

Comment:
We could just remove the async keyword here I think? If not, we should make that possible to remove the ignore lint above.